includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -57,9 +57,9 @@
 		 * @access private
 		 */
 		private function __construct() {
+			self::$per_page  = absint( ! empty( $_REQUEST['per_page'] ) ? $_REQUEST['per_page'] : self::$per_page );
```

Comment:
@raftaar1191 Using `$_REQUEST` is bad practice, please use appropriate HTTP method

Reply:
Thanks @ravinderk for pointing this out

Reply:
Fixed